notebooks/Advanced_Usage.ipynb

```diff
@@ -37,7 +37,11 @@
     "  * [Accessing callback parameters](#Accessing-callback-parameters)\n",
```

Comment:
<p>This cell does not reach the promised accuracy of 70%, maybe it would make sense to write an assertion so that such errors are caught directly?</p><p><br></p>

 _Reply via <a href='https://app.reviewnb.com/skorch-dev/skorch/pull/457/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
:+1: